pages/index.js

```diff
@@ -10,6 +11,8 @@ export default function Home() {
 
       <main>
         <h1 className="title">
+          <Link   href={{ pathname: '/api/hello'}}><a>my response</a></Link>
```

Comment:
Check this example on how to consume APIs: https://github.com/vercel/next.js/blob/canary/examples/api-routes/pages/index.js